src/plugins/tooltip/style.scss

```diff
@@ -0,0 +1,126 @@
+@use '../../setup/scss/variables' as *;
+@use '../../setup/scss/mixins' as *;
+@use '../../setup/scss/functions' as *;
+
+.c-tooltip {
+  $this: &;
+  $pointer-width: 6px;
+  $wrapper-padding: $pointer-width + 6px;
+
+  position: relative;
+  display: block;
+  cursor: pointer;
+
+  @at-root {
+    span#{&} {
+      display: inline-block;
+    }
+  }
+
+  &__anchor {
+    cursor: pointer;
+  }
+
+  &__tooltip-wrapper {
+    @include font($font-size--14);
+    @include z-index(front); // Was required for IE11.
+
+    display: none;
+    position: absolute;
+    opacity: 0;
+    transition: opacity $transition-duration--100;
+    pointer-events: none;
+    max-width: 80vw;
+
+    @include media(sm) {
+      max-width: rem(400px);
```

Comment:
todo: please remove the `rem()` function from this file. This was project specific.